### ISSUE TITLE
[Bug]: Unable to visualize graph queries on current redis-stack:6.2.4-v3

### ISSUE THREAD

**Issue by Kralizek** [RedisInsight/RedisInsight#1263]
**Steps to reproduce** (How to reproduce what you found step by step)
- pull `redis-stack:latest`
- create a container with this image: `docker run -d --name redis-stack -p 6379:6379 -p 8001:8001 redis/redis-stack:latest`
- go to redis insight: http://localhost:8001
- create a node `GRAPH.QUERY dungeon "CREATE ()"`

GRAPH VIEW will return the following error: `/static/plugins/redisgraph/dist/index.js not found. Check if it has been renamed or deleted and try again.`

TEXT VIEW will work as intended:
```
1) 1) "Nodes created: 1"
   2) "Cached execution: 0"
   3) "Query internal execution time: 0.261609 milliseconds"
```

**Actual behavior** (A short description of what you found)
When executing GRAPH queries, a JS error occurs.

e.g., `GRAPH.QUERY dungeon "MATCH (r) RETURN r"`

**Expected behavior** (A short description of what you expected to find)
The graph can be visualized.

**Screenshots** (Paste or drag-and-drop a screenshot or a link to a recording)
<img width="764" alt="image" src="https://user-images.githubusercontent.com/2458439/194435510-246f91a5-5d07-40d2-b51f-372570738d33.png">

**Additional context** (Operating system, version of RedisInsight, Redis database version, Redis module version, database type, connection type, logs, or any other information)
OS: Windows
Redis product versions as packaged in redis-stack:6.2.4-v3 (as of writing, this is `latest`)

**Comment by ViktarStarastsenka** [comment]
hi @Kralizek , thank you for reporting this, we will investigate the issue.

**Comment by Kralizek** [comment]
@ViktarStarastsenka btw, is there a way in Redis Insights to open a window where cypher queries can be written and executed without the whole `GRAPH.QUERY` ceremony?

**Comment by ViktarStarastsenka** [comment]
currently, no, you need to write GRAPH.QUERY and specify the graph.
I recommend using Workbench - we have the Cypher editor there that can be opened inside the query (with syntax highlighting and autocompletion).
could you please describe your use case so I can understand it better?

**Comment by Kralizek** [comment]
I'm experimenting with Graph DB for the first time and for someone coming from SQL Server Management Studio, it feels weird having to write _GRAPH.QUERY <dbname> "query query query"_ every time.

I know that by pressing SHIFT+SPACE a Cypher specific window opens, but it would be cool to have direct access to it.

**Comment by ViktarStarastsenka** [comment]
hi @Kralizek , the issue should be fixed with 2.12.0 (released today), please check.
